draft-ietf-tls-tls13.md

```diff
@@ -2434,10 +2524,13 @@ identifier
 {: br}
 
 When the client offers a PSK cipher suite, it MUST also supply a
-PreSharedKeyExtension to indicate the PSK(s) to be used. If no
-such extension is present, the server MUST NOT negotiate
-a PSK cipher suite. If no suitable identity is present, the server
-MUST NOT negotiate a PSK cipher suite.
+PreSharedKeyExtension to indicate the pre-shared key(s) to be used.
+Servers negotiating TLS 1.3 for a ClientHello without this extension yet
+offering any PSK cipher suite MUST send a "missing_extension"
+alert message and close the connection.
+If no suitable identity is present, the server MUST NOT negotiate
+a PSK cipher suite and MAY respond with an "unknown_psk_identity"
+alert message.
```

Comment:
If you mean this:
https://tools.ietf.org/html/rfc4279#section-2
>   If the server does not recognize the PSK identity, it MAY respond
>   with an "unknown_psk_identity" alert message.  Alternatively, if the
>   server wishes to hide the fact that the PSK identity was not known,
>   it MAY continue the protocol as if the PSK identity existed but the
>   key was incorrect: that is, respond with a "decrypt_error" alert.

That's still here. This section is why I have "MAY respond with an "unknown_psk_identity"", rather than a "SHOULD" or "MUST".